samples/bluetooth/peripheral_hr/sample.bluetooth.peripheral_hr_rv32m1_vega_ri5cy fails to build on  rv32m1_vega_ri5cy

`./scripts/sanitycheck -p rv32m1_vega_ri5cy -s samples/bluetooth/peripheral_hr/sample.bluetooth.peripheral_hr_rv32m1_vega_ri5cy`

Gets

```
/home/galak/git/zephyr/subsys/bluetooth/controller/ll_sw/openisa/lll/lll_conn.c: In function 'isr_done':
/home/galak/git/zephyr/subsys/bluetooth/controller/ll_sw/openisa/lll/lll_conn.c:600:5: error: 'struct event_done_extra' has no member named 'slave'
  600 |    e->slave.start_to_address_actual_us =
      |     ^~
/home/galak/git/zephyr/subsys/bluetooth/controller/ll_sw/openisa/lll/lll_conn.c:602:5: error: 'struct event_done_extra' has no member named 'slave'
  602 |    e->slave.window_widening_event_us =
      |     ^~
/home/galak/git/zephyr/subsys/bluetooth/controller/ll_sw/openisa/lll/lll_conn.c:604:5: error: 'struct event_done_extra' has no member named 'slave'
  604 |    e->slave.preamble_to_addr_us = preamble_to_addr_us;
      |     ^~
gmake[2]: *** [zephyr/subsys/bluetooth/controller/CMakeFiles/subsys__bluetooth__controller.dir/build.make:343: zephyr/subsys/bluetooth/controller/CMakeFiles/subsys__bluetooth__controller.dir/ll_sw/openisa/lll/lll_conn.c.obj] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:3187: zephyr/subsys/bluetooth/controller/CMakeFiles/subsys__bluetooth__controller.dir/all] Error 2
gmake: *** [Makefile:104: all] Error 2
```

--- a/subsys/bluetooth/controller/ll_sw/openisa/lll/lll_conn.c
+++ b/subsys/bluetooth/controller/ll_sw/openisa/lll/lll_conn.c
@@ -597,11 +597,11 @@ static void isr_done(void *param)
 				addr_us_get(0);
 #endif /* !CONFIG_BT_CTLR_PHY */
 
-			e->slave.start_to_address_actual_us =
+			e->drift.start_to_address_actual_us =
 				radio_tmr_aa_restore() - radio_tmr_ready_get();
-			e->slave.window_widening_event_us =
+			e->drift.window_widening_event_us =
 				lll->slave.window_widening_event_us;
-			e->slave.preamble_to_addr_us = preamble_to_addr_us;
+			e->drift.preamble_to_addr_us = preamble_to_addr_us;
 
 			/* Reset window widening, as anchor point sync-ed */
 			lll->slave.window_widening_event_us = 0;

